android/java/org/chromium/chrome/browser/crypto_wallet/util/ImageLoader.java

```diff
@@ -0,0 +1,82 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.crypto_wallet.util;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.text.TextUtils;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.Priority;
+import com.bumptech.glide.RequestBuilder;
+import com.bumptech.glide.load.engine.DiskCacheStrategy;
+import com.bumptech.glide.load.resource.bitmap.CenterInside;
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners;
+import com.bumptech.glide.load.resource.gif.GifDrawable;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ImageLoader {
+    private static final List<String> ANIMATED_lIST = Arrays.asList(".gif");
+
+    public static void loadNft(
+            String url, ImageView imageView, Context context, boolean isCircular) {
+        if (isAnimated(url)) {
+            loadGif(url, imageView, context, isCircular);
+        } else {
+            loadImg(url, imageView, context, isCircular);
```

Comment:
The two methods differ only for `asGif` if I'm not mistaken. 
Then they can converge into one and remove the `asGif`, the GIF will work correctly.

`asGif` is used for handling the `GifDrawable`, useful for extra manipulations.

Reply:
Cool, Likely due to the updates with Glide 3.0.
done!